src/rating/rating.ts

```diff
@@ -144,9 +144,16 @@ export class NgbRating implements ControlValueAccessor,
 
   handleBlur() { this.onTouched(); }
 
-  handleClick(value: number) { this.update(this.resettable && this.rate === value ? 0 : value); }
+  handleClick(value: number) {
+    if (!this.readonly && !this.disabled) {
+      this.update(this.resettable && this.rate === value ? 0 : value);
+    }
+  }
 
   handleKeyDown(event: KeyboardEvent) {
+    if (this.readonly || this.disabled) {
```

Comment:
This check is not covered by tests:

![Screen Shot 2020-02-07 at 11 17 17](https://user-images.githubusercontent.com/8074436/74021207-729ef500-499b-11ea-9bf6-b36ced9d433b.png)